documentation/Migration.md

```diff
@@ -1,9 +1,122 @@
 
 If you have an existing application that uses the older Node.js Azure SDK packages and you're interested in updating your application to use the newer JavaScript Azure SDK packages, then the good news is that there is very little for you to do. Here's the things that have changed with this new set of SDKs:
 
+## TL;DR
+
+<!-- markdownlint-disable MD033 -->
+<table>
+  <tr>
+    <td colspan="2">
+      <p>
+        Runtime package name was incorporated into <code>@azure</code> NPM scope and "-js" suffix was added. <a href="#imports">Read more</a>.
+      </p>
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <pre lang="typescript">
+import * as msRest from "ms-rest";
+      </pre>
+    </td>
+    <td>
+      <pre lang="typescript">
+import * as msRestJs from "@azure/ms-rest-js";
+      </pre>
+    </td>
+  </tr>
+  <tr>
+    <td colspan="2">
+      <p>
+        <code>ms-rest-azure</code> package was moved into <code>@azure</code> NPM scope and authentication related code was extracted into two separate packages. <a href="#authentication">Read more</a>.
+      </p>
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <pre lang="typescript">
+import * as msRestAzure from "ms-rest-azure";
+      </pre>
+    </td>
+    <td>
+      <pre lang="typescript">
+import * as msRestAzureJs from "@azure/ms-rest-azure-js";
+import * as msRestNodeAuth from "@azure/ms-rest-nodeauth";
+import * as msRestBrowserAuth
+                        from "@azure/ms-rest-browserauth";
+      </pre>
+    </td>
+  </tr>
+ <tr>
+    <td colspan="2">
+      <p>
+        In order to reduce bundle size, all necessary properties were extracted into "Context" (e.g. <code>StorageManagementClientContext</code>) object. It does not enable performing any operations. <a href="#context-types">Read more</a>.
+      </p>
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <pre lang="typescript">
+class StorageManagementClient {
+  subscriptionId: string;
+  <br />
+  storageAccounts: new StorageAccounts(this);
+  blobContainers: new BlobContainers(this);
+}
+      </pre>
+    </td>
+    <td>
+      <pre lang="typescript">
+class StorageManagementClientContext {
+  subscriptionId: string;
+}
+<br />
+class StorageManagementClient
+      extends StorageManagementClientContext {
+  storageAccounts: new StorageAccounts(this);
+  blobContainers: new BlobContainers(this);
+}
+      </pre>
+    </td>
+  </tr>
+ <tr>
+    <td colspan="2">
+      <p>
+        All operations are still grouped together by categories. They take a dependency on "Context" object instead of whole client. <a href="#context-types">Read more</a>.
```

Comment:
Operations are still grouped together...
...on "Context" object instead of the entire client.